docs/api-reference/carto/overview.md

```diff
@@ -44,6 +49,35 @@ function App({viewState}) {
 }
 ```
 
+## Usage CARTO Cloud Native
+
+```js
+import DeckGL from '@deck.gl/react';
+import {CartoLayer, setDefaultCredentials, MAP_TYPES, API_VERSIONS} from '@deck.gl/carto';
+
+setDefaultCredentials({
+  apiVersion: API_VERSIONS.V3,
+  apiBaseUrl: 'https://gcp-us-east1.api.carto.com', 
+  accessToken: 'XXX',
+});
+
+function App({viewState}) {
+  const layer = new CartoLayer({
+    type: MAP_TYPES.QUERY,
+    connection: 'bigquery'
+    data: 'SELECT * FROM cartobq.testtables.points_10k',
+    pointRadiusMinPixels: 2,
+    getLineColor: [0, 0, 0, 200],
+    getFillColor: [238, 77, 90],
+    lineWidthMinPixels: 1
+  })
+
+  return <DeckGL viewState={viewState} layers={[layer]} />;
+}
+```
+
+**CARTO Cloud Native** is currently available only in a private beta. If you want to test it, please contact us at [support@carto.com](mailto:support@carto.com?subject=Access%20to%20Cloud%20%Native%20%API%20(v3)).
```

Comment:
- Maybe start this section with a brief sentence about what CARTO Cloud native is.
- Move this line before the example
- Add `>` first to this line to highlight it in a "yellow warning box"

```suggestion
> **CARTO Cloud Native** is currently available only in a private beta. If you want to test it, please contact us at [support@carto.com](mailto:support@carto.com?subject=Access%20to%20Cloud%20%Native%20%API%20(v3)).
```